terminal.go

```diff
@@ -838,6 +839,7 @@ func (g *LightningTerminal) shutdown() error {
 		}
 	}
 
+	g.sessionRpcServer.stop()
```

Comment:
Doesn't' look to be protected against a double chan close.

Reply:
cool, added a `sync.Once` to protect against that